### ISSUE TITLE
Integration of Consumers, Producers & Subscriptions

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#97]
## User story
1. As a user
2. I want to have a UI integrated with the backend
3. So that I can see all information about consumers, prodcuers, subscriptions and topics in the Apache Pulsar UI.

## Acceptance criteria
Integrate Backend Endpoint for the following sections:
* [ ] Subscription Pop-Up
* [ ] Consumer Pop-Up
* [ ] Producer Pop-Up

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
